cypress/support/api/requests/ProductType.js

```diff
@@ -143,3 +143,22 @@ export function setProductTypeAsDigital(productTypeId, isDigital = true) {
    }`;
   return cy.sendRequestWithQuery(mutation);
 }
+
+export function assignAttribute(
+  productTypeId,
+  attributeId,
+  attributeType = "VARIANT"
+) {
+  const mutation = `mutation{
+    productAttributeAssign(productTypeId:"${productTypeId}" operations:{
```

Comment:
```suggestion
    productAttributeAssign(productTypeId:"${productTypeId}", operations:{
```
It was okay without the comma? Alternatively the formatting is off